docs/development/accessibility.md

```diff
@@ -796,6 +796,24 @@ Check how your suggested element fits into our semantic structures.
 8.	Always use the language attribute on the HTML element. Screen readers change pronunciation automatically. (3.1.1)
 9.	Indicate the language for each chunk of text (3.1.2). Using the language attribute on the HTML element. Screen readers change pronunciation automatically. 
 10.	Functional and informative components, i.e. labels and icons, must have consistent labels, aria-labels and alt-texts. 
+##Landmark Roles
+* Banner
+    * For the Header of the UI Component [*Standard Page*](../../src/UI/Component/Layout/Page/Factory.php), where Logo and Title are placed, the HTML tag < header > is used to be identified as the ARIA Landmark Role *Banner*.
```

Comment:
This goes for all the following bullet points:
I read the passive tense in this sentence as "it is enough to put a ``<header>`` tag around it to make sure that automatically a rolle *Banner* is added". Is this really what you want to say, or should it be "...is used and a Landmark Role *Banner* is added."?

Reply:
The former is correct. It is "enough" to use semantic HTML here. The Screen Reader reads the Landmark Role automatically. Saying that the Landmark Role is _added_ would be wrong IMO because the Landmark Role is not added in the HTML, but the Screenreader itself identifies it.
But I am open to change the wording if it can be done more comprehensible.